### DISCUSSION TITLE
How strong is the drawing?

### DISCUSSION THREAD

**Discussion by Tibitron** [keiwando/toonsquid#162] [Q&A]
My question is how strong is Toonsquid as a drawing program? Can it replace procreate and be a correct drawing and animating program at the same time?

**Comment by Cat-Butler** [comment]
Better than the paint/draw features in some other frame-by-frame animation apps, but certainly not a feature-for-feature Procreate replacement.
For backgrounds with lots of detail/texture, for example, it might be better to paint in Procreate and import, or even (for now, at least) for background perspective assistance.
Doesn't yet have anywhere near the brush ecosystem of Procreate, and a lesser selection of built-in brushes.

I think it's reasonable for a basic paint/drawing program (good enough for a lot of animation, you don't want to be spending a couple of hours re-painting each frame), has useful vector functionality, is significantly better for animation features, but I wouldn't put it at Procreate level for painting/drawing.
